.github/workflows/release-kotlin-bindings-nix.yml

```
@@ -0,0 +1,89 @@
+        uses: actions/checkout@v4
+      - name: Download artifacts
+        uses: actions/download-artifact@v4
+        with:
+          path: bindings_ffi/jniLibs
+      - name: Build archive
+        working-directory: bindings_ffi
+        run: |
+          zip -r LibXMTPKotlinFFI.zip jniLibs
+      - name: Get short SHA
+        id: slug
+        run: echo "sha7=$(echo ${GITHUB_SHA} | cut -c1-7)" >> "$GITHUB_OUTPUT"
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:19: Double quote to prevent globbing and word splitting

(shellcheck)